backend-test/integration/auth.test.js

```diff
@@ -1,66 +0,0 @@
-/* eslint no-undef: 0 */
```

Comment:
Why was this file deleted?

Reply:
I moved all the auth tests there to user.test.js, since I also wrote tests for the user profile there